symphony-bdk-core/src/main/java/com/symphony/bdk/core/client/ApiClientFactory.java

```diff
@@ -95,6 +96,22 @@ public ApiClient getSessionAuthClient() {
     return buildClientWithCertificate(this.config.getSessionAuth().getBasePath() + "/sessionauth");
```

Comment:
I'd change this line by: 
```java
return buildClientWithCertificate(this.config.getSessionAuth().getBasePath() + "/sessionauth", botConfig.getCertificatePath(), botConfig.getCertificatePassword());
```
and remove method `buildClientWithCertificate(String basePath)`